tests/raft_log_replication.py

```diff
@@ -626,20 +527,6 @@ async def _unittest_raft_log_replication() -> None:
   | 0          | 4          | 5          | 6          | 7           |     Log term
```

Comment:
Interesting, how the terms in the comment are all different, and in the test they're all the same (1).

Reply:
Solved in branch fix-term commit 8d01fa6